wlauto/workloads/geekbench/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java

```diff
@@ -160,7 +156,7 @@ public void shareResults() throws Exception {
         getUiDevice().pressMenu();
         UiObject shareButton = new UiObject(selector.text("Share")
                                                     .className("android.widget.TextView"));
-        shareButton.waitForExists(WAIT_TIMEOUT_1SEC);
+        shareButton.waitForExists(WAIT_TIMEOUT_20MIN);
```

Comment:
Why does the timeout need to be this long here? This is basically waiting for the app to generate the json data dump?